entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityQueryServiceImpl.java

```diff
@@ -183,11 +172,9 @@ public EntityQueryServiceImpl(
         entityCounterMetricSender,
         new EntityFetcher(entitiesCollection, DOCUMENT_PARSER),
         chunkSize,
-        queryAggregationEnabled,
         maxEntitiesToDelete);
   }
 
-  @VisibleForTesting
```

Comment:
The above constructor itself is calling this constructor: https://github.com/hypertrace/entity-service/pull/229/files#diff-0aad350d7986d390c6ff43b5f9074b244ea7b1a7e1baef820a4eb4b70a58f5b3L178

Apart from the test files, it is being called at this place only.
<img width="1296" alt="image" src="https://user-images.githubusercontent.com/97032782/227118430-c326938f-a077-4c09-b459-e808badbdefa.png">